db/schema.rb

```diff
@@ -15,13 +15,13 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "collections", id: :serial, force: :cascade do |t|
```

Comment:
why is the ID being removed from all these tables?

Reply:
This is a difference between Chris and my local environments, I think.  I would prefer not to remove it because I think we should specify it now that the default has changed.

Reply:
yea, I think this change needs to be rolled back, we should keep the serial

Reply:
agree